Simplenote/SpinnerViewController.swift

```diff
@@ -0,0 +1,68 @@
+import UIKit
+
+class SpinnerViewController: UIViewController {
+    private var alertView: UIView!
+    private var activityIndicator: UIActivityIndicatorView!
```

Comment:
Nipticky: since you're not using nibs, you can drop the Implicitly Unwrapped Optionals:

```
private lazy var alertView = UIView()
private lazy var activityIndicator = UIActivityIndicatorView()
```